index.html

```diff
@@ -1,6 +1,17 @@
-<html>
-    <head>
-        <script src="https://cdn.ethers.io/lib/ethers-5.2.umd.min.js"
+<!doctype html>
+<html lang="en">
+  <head>
+    <title>Title</title>
```

Comment:
If we're adding a title, lets set it to something meaningful - perhaps `Simple Temple Balance Checker`? Open to thoughts on this